docs/replication/group-replication-system-variables.md

```diff
@@ -0,0 +1,47 @@
+# Group replication system variables
+
+
+| group_replication_auto_evict_timeout |
+|---|
+| group_replication_flow_control_mode |
+|  |
+
+## group_replication_auto_evict_timeout
+
+The variable is in [tech preview](../glossary.md#tech-preview) mode. Before using the variable in production, we recommend that you test restoring production from physical backups in your environment, and also use the alternative backup method for redundancy.
+
+| Option | Description |
+|---|---|
+| Introduced | 8.0.30-22 |
+| Command-line |--group-replication-auto-evict-timeout  |
+| Dynamic | Yes |
+| Scope | Global |
+| Type | Integer |
+| Default value | 0 |
+| Maximum Value | 65535  |
+| Unit | seconds |
+
+The value can be changed while Group Replication is running. The change takes effect immediately. Every node in the group can have a different timeout value, but, to avoid unexpected exits, we recommend that all nodes have the same value.
+
+The variable specifies a period of time in seconds before a node is automatically evicted if the node exceeds the [flow control threshold](group-replication-flow-control.md). The default value is 0, which does not change the existing behavior. To set the timeout, change the value with a number higher than zero.
+
+In single-primary mode, a single server ignores a change in the timeout value.
```

Comment:
"a single server" - no, the primary node. (and it ignores the timeout, not the change)

Reply:
updated.